var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -26,11 +26,11 @@ class Vtk(CMakePackage):
     version('9.0.3', sha256='bc3eb9625b2b8dbfecb6052a2ab091fc91405de4333b0ec68f3323815154ed8a', preferred=True)
     version('9.0.1', sha256='1b39a5e191c282861e7af4101eaa8585969a2de05f5646c9199a161213a622c7')
     version('9.0.0', sha256='15def4e6f84d72f82386617fe595ec124dda3cbd13ea19a0dcd91583197d8715')
-    version('8.2.0', sha256='34c3dc775261be5e45a8049155f7228b6bd668106c72a3c435d95730d17d57bb')
-    version('8.1.2', sha256='0995fb36857dd76ccfb8bb07350c214d9f9099e80b1e66b4a8909311f24ff0db')
+    version('8.2.0', sha256='34c3dc775261be5e45a8049155f7228b6bd668106c72a3c435d95730d17d57bb', preferred=True)
+    version('8.1.2', sha256='0995fb36857dd76ccfb8bb07350c214d9f9099e80b1e66b4a8909311f24ff0db', preferred=True)
```

Comment:
Why this choice of preferred versions (e.g. 8.0.1 preferred to 8.1.1)?

Reply:
It's mostly to help guide the concretizer so you don't end up with automatically selected middle versions.  I was having issues specifying `depends_on('vtk@8.1.0:8.1')` and having it resolve to `8.1.1` instead of `8.1.2`.  This solves that by putting all the last patch releases in a series first. However, it looks like I missed a few (I'll update accordingly)